server.go

```diff
@@ -49,15 +49,15 @@ type ServerConfig struct {
 // the configuration.
 func (c *ServerConfig) Validate() error {
 	catcher := grip.NewBasicCatcher()
-	catcher.NewWhen(c.TLS != nil && c.TLS.Certificates == nil, "tls config specified without certificates")
-	catcher.NewWhen(c.Handler == nil && c.App == nil, "must specify a handler or a gimlet app")
+	catcher.NewWhen(c.TLS != nil && c.TLS.Certificates == nil, "TLS config specified without certificates")
+	catcher.NewWhen(c.Handler == nil && c.App == nil, "must specify a handler or an app")
 	catcher.NewWhen(c.Handler != nil && c.App != nil && !c.handlerGenerated, "can only specify a handler or an app")
 	catcher.NewWhen(c.Address == "", "must specify an address")
-	catcher.ErrorfWhen(c.Timeout < time.Second, "must specify timeout greater than a second, '%s'", c.Timeout)
-	catcher.ErrorfWhen(c.Timeout > 10*time.Minute, "must specify timeout less than 10 minutes, '%s'", c.Timeout)
+	catcher.NewWhen(c.Timeout < time.Second, "must specify timeout greater than a second")
```

Comment:
Should this be `greater than or equal to a second`?

Reply:
Done.